concent_api/verifier/decorators.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+
+from django.conf import settings
+
+from core.constants import VerificationResult
+from core.tasks import verification_result
+from verifier.exceptions import VerificationError
+from verifier.utils import clean_directory
+
+
+def handle_verification_errors(task):
+    @wraps(task)
+    def wrapper(*args, **kwargs):
+        try:
+            return task(*args, **kwargs)
+        except VerificationError as exception:
+            verification_result.delay(
+                exception.subtask_id,
```

Comment:
You can simplify the `VerificationError` and take `subtask_id` from `*args`.

Reply:
It would be a bit risky or not elegant:
I would need to check both `*args` and `**kwargs` as I have no warranty that `subtaski_id` will be passed this or other way. checking both seems a bit unelegant, so I would be for feeding it explicitly to `VerificationError`

Reply:
Risky - I agree, not elegant - I think opposite :) But please consider that in case of failure in task you pass the same parameter to `VerificationError`, not knowing if it is really `subtask_id`.